### PR TITLE
Update Signs to execute top/info commands

### PR DESCRIPTION
**Pull request by nordlead2005**
Updated signs to support Top (Fastest, Fastest Lap, Wins, Win Ratio, Runs), and Info commands. Made sign format consistent across types, and added laps to race join signs. Also removed warnings to make finding new errors/warnings easier.

To create a sign, you now enter:
 - race
 - race name
 - join/fastest/fastest lap/wins/win ratio/runs/info
 - optional lap count

Note, that join signs have a limit of 99 laps, 99/99 participants before it runs over the sign line length (unless invisible characters don't count, I've never tried programatically). I think this is reasonable, since I don't think most +1.13 servers ever have more than 100 people. Also, 100 laps is a serious marathon.

I wasn't sure what to do with the send(Top/Info)Message functions, so they live with the commands class. I thought about moving them to Util, but that didn't seem like the right location.

This was drivin by the fact that most people (my kids/family) don't want to learn the commands, this gives them an easy interface to the plugin.

### CODE REVIEW

**Comment by hornta** [comment]
@nordlead2005 Hi. Totally forgot this. Will check it ASAP.

**Comment by nordlead2005** [comment]
@hornta No problem. Was staring to wonder if you were doing well with the coronavirus being a thing and all.

File: src/main/resources/translations/Racing/english.yml [hunk 1]
```diff
@@ -107,7 +107,7 @@ commands:
     entry_fee_line: "§eEntry fee: §f<entry_fee>\n"
   race_top:
     types:
-      fastest: 'Fastest run for lap count'
+      fastest: 'Fastest run for <laps> laps'
```
Comment (hornta) [inline src/main/resources/translations/Racing/english.yml:110]: I guess it reads weird with "Fastest run for 1 laps" but that's just my OCD wanting to have 1 lap, 2 laps and so on.. but not necessary to change.

**Comment by hornta** [comment]
LGTM 👍

File: src/main/java/com/github/hornta/race/Racing.java [hunk 2]
```diff
@@ -46,6 +46,7 @@
   private Translations translations;
   private RacingManager racingManager;
   private ProtocolManager protocolManager;
+  @SuppressWarnings("unused")
```
[the remark this answers is not in the document]
Comment (nordlead2005) [reply src/main/java/com/github/hornta/race/Racing.java:49]: Yeah, I saw that and had the same thought. Surpressing the warning leaves it as it while reducing the clutter in my IDE.

File: src/main/resources/translations/Racing/english.yml (hunk 1 above)
Comment (nordlead2005) [reply src/main/resources/translations/Racing/english.yml]: yeah, it irked me as well and I ignored it, but it is fixed now.

Comment (nordlead2005) [reply src/main/resources/translations/Racing/english.yml]: actually, not resolved, I need to add a translation for plurals... be right back 😄

File: src/main/java/com/github/hornta/race/commands/CommandTop.java [hunk 3]
```diff
@@ -31,6 +31,7 @@ public void handle(CommandSender commandSender, String[] args, int typedArgs) {
   static public void sendTopMessage(CommandSender target, Race race, int laps, RaceStatType statType) {
     MessageManager.setValue("type", MessageManager.getMessage(statType.getKey()));
     MessageManager.setValue("laps", laps);
+    MessageManager.setValue("plural", (laps > 1) ? "s" : "");
```
Comment (hornta) [inline src/main/java/com/github/hornta/race/commands/CommandTop.java:34]: I guess this works for english but I'm not sure about other languages. This is quite hard to get right for all languages. What I use to do is to two separate translations: "laps_singular" and "laps_plural" and give them the values "lap" and "laps". Then you "fetch" those values using MessageManager.getMessage(MessageKey.LAP_SINGULAR or MessageKey.LAP_PLURAL) and pass them to the .setValue method. The swedish translation for laps is "varv" for both singular and plural. I believe this covers the majority of languages. I know there are some languages using three different forms even 😄

Comment (nordlead2005) [reply src/main/java/com/github/hornta/race/commands/CommandTop.java:34]: yeah, I realized this right after I put it there...

**Comment by hornta** [comment]
> @hornta No problem. Was staring to wonder if you were doing well with the coronavirus being a thing and all.

Im good thanks. We are working from home since 1.5 month ago. How r u?

File: src/main/java/com/github/hornta/race/commands/CommandTop.java (hunk 3 above)
Comment (nordlead2005) [reply src/main/java/com/github/hornta/race/commands/CommandTop.java]: I updated it and tested it out via command line and it looks correct now.

**Comment by nordlead2005** [comment]
> > @hornta No problem. Was staring to wonder if you were doing well with the coronavirus being a thing and all.
> 
> Im good thanks. We are working from home since 1.5 month ago. How r u?

We are doing well. My company transitioned to everyone working from home almost 3 weeks ago and we home school the kids. So, hopefully no one gets sick, and I can do this as long as required to stay safe (our customer is generally the government so no worry about running out of work since they'll just borrow/print more money 💰).

**Pull request merged**